### DISCUSSION TITLE
[Question]JPK_FA export

### DISCUSSION THREAD

**Discussion by pmuch78** [YetiForceCompany/YetiForceCRM#16875] [Feature improvement]
Is tghere a possibility to export invoices to JPK_FA format?

**Comment by mariuszkrzaczkowski** [comment]
not in the current version, https://github.com/YetiForceCompany/YetiForceCRM/discussions/16795

**Reply by pmuch78** [reply]
#16795 is about KSeF - for now I'm looking for a possibility to exchange data with my acoountant - it's pretty old official format.
